test/ng/directive/ngRepeatSpec.js

```diff
@@ -1424,15 +1424,58 @@ describe('ngRepeat animations', function() {
     };
   }));
 
+  beforeEach(function() {
```

Comment:
Tests should not rely on the order of items in the animation queue. This Jasmine matcher looks for a queued item with a given element text and matches against that item's event.